evaluators/ragas_evaluator.py

```
@@ -159,54 +161,115 @@ def _evaluate_ragas_sync(self, question: str, answer: str, ground_truth: str, co
+            
+            # 处理结果
+            relevancy_scores = []
+            correctness_scores = []
+            faithfulness_scores = []
+            context_precision_scores = []
+            context_recall_scores = []
+            
+            try:
+                scores_dict = result.scores
+                
+                # scores_dict 是列表格式，每个元素是一个字典
```

Ruff (0.12.2)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)